docs/en_us/platform_api/source/courses/blocks.rst

```diff
@@ -0,0 +1,350 @@
+.. _Courses API Blocks Resource:
+
+########################################
+Courses API Blocks Resource
+########################################
+
+With the Courses API **Blocks** resource, you can complete the
```

Comment:
I think we should remove the bold formatting here. Bold is for GUI labels, right? This could be something like:

With the ``/api/courses/v1/blocks`` resource, you can complete the